### PR TITLE
Add ECMWF ecFlow

### PR DESCRIPTION
**Pull request by dtip**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dtip** [comment]
@conda-forge/help-c-cpp This is ready for review, please.

The OSX CI timeout out after 6 hours somehow... but I've run it locally with `CONFIG=azure-osx-64-comp7 .circleci/run_docker_build.sh` and the build and tests work fine.

Could we merge despite the allegedly failing CI?

**Comment by dtip** [comment]
I'll do a little force push to get the CI to re-run but won't actually change anything.

**Comment by dtip** [comment]
Hi @conda-forge/staged-recipes could someone take a look at this please? Would be helpful to unblock some downstream work. OSX build is timing out but there seems to be nothing I can do about it!

File: recipes/ecflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - ecflow_server --help
+    - ecflow_standalone --help
+    - ecflow_ui_log -h
+
+about:
+  home: https://confluence.ecmwf.int/display/ECFLOW
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: Apache 2.0
```
Comment (chrisburr) [inline recipes/ecflow/meta.yaml]: For consistency with other recipes:
```suggestion
  license: Apache-2.0
```

File: recipes/ecflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - conda inspect objects -p $PREFIX ecflow  # [osx]
+    - ecflow_client --help
+    - ecflow_server --help
+    - ecflow_standalone --help
+    - ecflow_ui_log -h
+
+about:
+  home: https://confluence.ecmwf.int/display/ECFLOW
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
```
Comment (chrisburr) [inline recipes/ecflow/meta.yaml]: Can you delete the comments which are from the example

**Comment by chrisburr** [comment]
> OSX build is timing out but there seems to be nothing I can do about it!

This seems to be an Azure issue that has been intermittently appearing over the last couple of weeks. I just restarted it manually but there is also a bot command anyone can use to restart the build: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-restart-ci

**Comment by dtip** [comment]
Thanks Chris. I've tried restarting the build 3 times already so not convinced it'll help sadly

**Comment by chrisburr** [comment]
It seems you're right... If you deal with my above comments I'll merge and we can see if macOS is still an issue on the feedstock.

File: recipes/ecflow/meta.yaml (hunk 2 above)
Comment (dtip) [reply recipes/ecflow/meta.yaml]: done

**Comment by dtip** [comment]
Thanks @chrisburr :)

**Pull request merged**